helpers/setup.sh

```diff
@@ -136,6 +136,7 @@ gcloud services enable \
     cloudresourcemanager.googleapis.com \
     serviceusage.googleapis.com \
     compute.googleapis.com \
+    container.googleapis.com \
```

Comment:
If this API is not needed for the GCE deployment of Forseti, then might be better to put it under the conditional "if ON_GKE" instead of putting it here.

Reply:
Done